src/analysis/retail/paladin/holy/modules/apl.tsx

```diff
@@ -0,0 +1,81 @@
+import SPELLS from 'common/SPELLS';
+import { suggestion } from 'parser/core/Analyzer';
+import aplCheck, { Apl, build, CheckResult, PlayerInfo, Rule } from 'parser/shared/metrics/apl';
+import annotateTimeline from 'parser/shared/metrics/apl/annotate';
+import TALENTS from 'common/TALENTS/paladin';
+import * as cnd from 'parser/shared/metrics/apl/conditions';
+
+import { AnyEvent } from 'parser/core/Events';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+
+const inWings = cnd.buffPresent(TALENTS.AVENGING_CRUSADER_TALENT);
+
+const APL: Rule[] = [
```

Comment:
HoW doesn't want an APL for holy paladin in wowA

Reply:
oh accidental inclusion, will delete. was playing with this when AC was the build, not even relevant now.

Reply:
but FWIW i think the MW one adds value